pkg/client/query/sessionquerier.go

```diff
@@ -3,7 +3,7 @@ package query
 import (
```

Comment:
The changes in this file seem to be unsafe & incomplete.

Can you:
1. Delete unnecessary commented out things
2. Either add back the mutexes or use xsync
3. Explain (in the comments in the code, not the github response) why we don't need locks here?